frontend/src/metabase/palette/components/PaletteResultsList.tsx

```diff
@@ -22,17 +29,17 @@ interface RenderParams<T = PaletteActionImpl | string> {
 
 interface PaletteResultListProps {
   items: (PaletteActionImpl | string)[];
-  onRender: (params: RenderParams) => React.ReactElement;
+  onRender: (params: RenderParams) => ReactElement;
```

Comment:
nitpick: since we're here, can we rename this to `renderItem`? That would make more sense imo, because it's not a callback property.